CMakeLists.txt

```diff
@@ -592,12 +592,12 @@ else()
 # Add sources
 add_subdirectory(src)
 
-endif()
-
 install(
-    FILES LICENSE-MPL2.md LICENSE-MIT-libmv.md COPYING.md CONTRIBUTORS.md
-    DESTINATION ${CMAKE_INSTALL_DATADIR}/aliceVision
-    )
+        FILES LICENSE-MPL2.md LICENSE-MIT-libmv.md COPYING.md CONTRIBUTORS.md
```

Comment:
@fabiencastan I moved it inside the else branch (builiding AV) because if you only build the dependencies you don't want to install all these files that are related to AV

Reply:
if ok we should also fix it in centos deps as you don't have to do `make install` but just `make` for the dependencies

Reply:
yes, sounds good.